core.spec.md

```diff
@@ -380,7 +373,7 @@ Satisfies(requested, available) :
 
 ## Referencing versions and activating implementations
 
-Schema documents MUST reference a feature version which supports all the schema elements and behaviors required by the document. As a practical matter, authors should generally prefer to reference the [lowest](#sec-Ordering) versions which can [satisfy](#sec-Satisfaction) their requirements, as this imposes the fewest constraints on processors and is thus more likely to be supported. Authors MAY choose a higher version which they believe has sufficient support.
+Schema documents MUST reference a feature version which supports all the schema elements and behaviors required by the document. As a practical matter, authors will generally prefer to reference a version they have reason to believe is supported by the most processors; depending on context, this might be an old stable version with a low major version, or a new less-deprecated version with a large major version.
```

Comment:
Yeah, this is in line with me thinking that offering specific suggestions based on ordering seemed speculative, I think. Should we just remove the entire section or something? Or leave in one sentence maybe somewhere else?

Reply:
I could see this sentence being transformed a bit to provide example / be more informative than speculative.

>Practically speaking, a spec author might prefer to reference a `@core` version which they have reason to believe is supported by most processors; depending on the requirements of the spec, this might be an old stable version with a low major version, or a new less-deprecated version with a large major version.

In re-reading this, I'm not actually sure it makes sense the way it talks about low vs. large major versions, right? Generally speaking, the way an author writes their spec will sorta dictate the major version that applies I think?

Reply:
Ah, maybe this isn't clear: this section is (I think!) about people writing schemas, not people writing specs.  So it's trying to tell you "given a choice between referencing `join` v2.3 and `join` v3.5, you should prefer... um... well honestly it probably depends"